methods.py

```diff
@@ -487,7 +487,7 @@ def find_visual_c_batch_file(env):
     from SCons.Tool.MSCommon.vc import get_default_version, get_host_target, find_batch_file
 
     version = get_default_version(env)
-    (host_platform, target_platform,req_target_platform) = get_host_target(env)
+    (host_platform, target_platform,_) = get_host_target(env)
```

Comment:
I know it was already missing, but can you add a space after `,`?

Reply:
Woops, sorry about that. Fixed.